app/javascript/elm/src/Data/SelectedSession.elm

```diff
@@ -122,23 +118,40 @@ view session heatMapThresholds linkIcon toMsg =
         [ p [ class "single-session-name" ] [ text session.title ]
         , p [ class "single-session-username" ] [ text session.username ]
         , p [ class "single-session-sensor" ] [ text session.sensorName ]
-        , div []
-            [ div [ class "single-session-avg-color", class <| Data.Session.classByValue (Just session.average) heatMapThresholds ] []
-            , span [] [ text "avg. " ]
-            , span [ class "single-session-avg" ] [ text <| String.fromInt <| round session.average ]
-            ]
-        , div [ class "session-numbers-container" ]
-            [ div [ class "single-min-max-container" ]
-                [ div [ class "single-session-color", class <| Data.Session.classByValue (Just session.min) heatMapThresholds ] []
-                , span [] [ text "min. " ]
-                , span [ class "single-session-min" ] [ text <| String.fromFloat session.min ]
-                ]
-            , div [ class "single-min-max-container" ]
-                [ div [ class "single-session-color", class <| Data.Session.classByValue (Just session.max) heatMapThresholds ] []
-                , span [] [ text "max. " ]
-                , span [ class "single-session-max" ] [ text <| String.fromFloat session.max ]
-                ]
-            ]
+        , case session.selectedRangeMeasurements of
+            [] ->
+                div [] []
```

Comment:
@gitjul How can I style this so that it's the same size as the div on line 136?

Reply:
is this still an issue?

Reply:
No, I styled this (but not responsive).